shoreman.sh

```diff
@@ -28,14 +28,20 @@ usage() {
 # to stdout.
 log() {
   local index="$2"
-  # Bash colors start from 31 up to 37. We calculate what color the process
-  # gets based on its index.
-  local color="$((31 + (index % 7)))"
+  local format="%s %s\t| %s"
+
+  # We add colors when output is a terminal. `SHOREMAN_COLORS` can override it.
+  if [ -t 1 -o "$SHOREMAN_COLORS" == "always" ] \
+     && [ "$SHOREMAN_COLORS" != "never" ]; then
+    # Bash colors start from 31 up to 37. We calculate what color the process
+    # gets based on its index.
+    local color="$((31 + (index % 7)))"
+    format=$(printf "\033[0;%sm%%s %%s\t|\033[0m %%s" "$color")
```

Comment:
Why not just drop `$color` right into where it needs to go? It removes the need to use `printf`.